flowauth/backend/flowauth/token_management.py

```diff
@@ -255,7 +264,7 @@ def generate_token(
 
     Examples
     --------
-    >>> generate_token(flowapi_identifier="TEST_SERVER",username="TEST_USER",secret="SECRET",lifetime=datetime.timedelta(5),claims={"daily_location":{"permissions": {"run":True},)
+    >>> generate_token(flowapi_identifier="TEST_SERVER",username="TEST_USER",private_key="SECRET",lifetime=datetime.timedelta(5),claims={"daily_location":{"permissions": {"run":True},)
```

Comment:
The example `private_key="SECRET"` gives the impression that the private key can be an arbitrary secret string. Instead, I'd prefer to give some indication about valid formats if possible, or at least make it clear that arbitrary strings won't work.